app/views/devise/passwords/new.html.haml

```diff
@@ -0,0 +1,13 @@
+= content_for :page_title_prefix, "Reset password"
```

Comment:
I notice elsewhere we use i18n for page titles. Should we be using one here (and presumably retrospectively for Log In?) as per [ADR 0011](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/2f1b3e2da49768074411df063d9a698d40a588e9/doc/architecture/decisions/0011-use-govuk-design-system-form-builder.md#L34-L35)? (apologies, did not notice the i18n stuff until today, and now having poked about, am more confused not less!)